### DISCUSSION TITLE
비둘기호, 통일호, 무궁화호 객차 등장년도 재조정 필요

### DISCUSSION THREAD

**Discussion by SerpensNebula** [KoreanGRF/KoreanTrainSet#299] [건의(Suggestions)]
https://github.com/KoreanGRF/KoreanTrainSet/issues/163

옛날에 진행됐던 논의에서 확장된 내용이긴 합니다만, 통일호에 적용된 잣대를 다른 객차에도 적용하면 다른 객차들도 등장년도를 다시 조정해야 합니다.

이유는 여러가지가 있습니다.

**첫번째**로는 통일호의 상위 등급인 무궁화호가 게임상으로 1974년에 먼저 등장하는 것이 이상하고, 

![제목 없음](https://user-images.githubusercontent.com/75788864/138886725-4109cd8f-5635-4be8-9ca0-0b4bd72f9ce4.png)
**두번째**로는 통일호는 1984년에 등장하게 되어있는데, 엄연히 1960년대부터 도입됐던 객차도 개조목록으로 지원하고 있습니다. https://github.com/KoreanGRF/KoreanTrainSet/issues/134

**세번째**로는 통일호만 84년 등장인 점입니다. 이름은 같지만 계통은 다르니 통일호를 84년 등장하게 한 위 이슈의 잣대로는, 무궁화호도 1984년에 신설됐으니 통일호와 똑같이 1984년으로 등장년도를 미뤄야 하고, 비둘기호는 1963년 이전 3등객차들은 분리신설하거나 https://github.com/KoreanGRF/KoreanTrainSet/issues/275 일제강점기 객차로 옮겨야 합니다. 

**네번째**로는 계통과 등급의 이름만 바뀌었지, 역할은 실제로는 같았거나 연속적으로 변화했던 점입니다.
![제목 없음 01](https://user-images.githubusercontent.com/75788864/138887561-c114764b-a5df-4539-941d-56207a96517d.png)
http://info.korail.com/mbs/www/subview.jsp?id=www_020102000000

|현재|등장년도|
|-|-|
|비둘기호|1927|-|
|통일호|1984|
|무궁화호|1974|
|새마을호|1984|

|**1안**|-|-|
|-|-|-|
|**현재 통일호 기준을 다른 객차 모두에게 적용할 경우**|등장년도|비고|
|비둘기호|1963|63년 이전 객차들을 분리신설해야 함|
|통일호|1984|1984년 이전 객차들을 완행, 급행 이름으로 분리신설해야 함|
|무궁화호|1984|1984년 이전 객차들을 완행, 급행 이름으로 분리신설해야 함|
|새마을호|1984 (1974)|현상유지 (괄호 안은 관광호 https://github.com/KoreanGRF/KoreanTrainSet/issues/241 가 구현될 경우)|
|관광호|1969|신설, 관광호 https://github.com/KoreanGRF/KoreanTrainSet/issues/241 가 구현될 경우|

|**2안**|-|-|
|-|-|-|
|**이전 60년대 등장 때의 기준으로 되돌릴 경우**|등장년도|비고|
|비둘기호|1927 (1938)|현상유지 (괄호 안은 일제강점기 객차 https://github.com/KoreanGRF/KoreanTrainSet/issues/275 가 구현된 후 재조정)|
|통일호|1963|개조 목록의 첫 연도 기준으로|
|무궁화호|1970(1977)|개조 목록의 첫 연도 기준(괄호 안은 https://github.com/KoreanGRF/KoreanTrainSet/issues/332 가 구현된 후 재조정) |
|새마을호|1984 (1969)|현상유지 (괄호 안은 관광호 https://github.com/KoreanGRF/KoreanTrainSet/issues/241 가 구현된 후 재조정)|

이 문제는 다른 기관차나 동차, 전철들은 도입년도나 성능별로 나뉘어져있는데, 객차는 혼자 여객등급별로 나뉘어져있는 태생적인 한계로 인한 어쩔 수 없는 혼란으로 보입니다. 그러나 최소한 통일호가 다른 객차들과 다른 잣대를 적용받아 등장년도가 혼자 뚝 떨어져있는 불합리한 면은 고쳐야 하는게 맞다고 봅니다.

**Comment by SerpensNebula** [comment]
새마을호는 현재 https://github.com/KoreanGRF/KoreanTrainSet/issues/241 직각형 객차가 구현되어 있지 않기 때문에, 1986년 등장이 맞습니다. 

관광호의 직각형 객차가 구현되면 관광호 등장년도인 1969년 혹은 새마을호 이름 등장년도인 1974년으로 당겨야 합니다.

**Comment by JukjeonWani** [comment]
현재 무궁화호 객차의 개조목록을 보면 일부도색이 통일호 객차와 겹치는 것으로 알고 있습니다. 제 개인적인 생각으로는 2안으로 가되 통일호 객차와 중복되는 도색을 무궁화호 객차에서 빼야 한다고 생각합니다

**Comment by telk5093** [comment]
> 현재 무궁화호 객차의 개조목록을 보면 일부도색이 통일호 객차와 겹치는 것으로 알고 있습니다. 제 개인적인 생각으로는 2안으로 가되 통일호 객차와 중복되는 도색을 무궁화호 객차에서 빼야 한다고 생각합니다

현재는 딱히 그럴 생각이 없고, 말씀하신 건은 본 이슈와 무관한 내용입니다.

**Comment by SerpensNebula** [comment]
> 현재 무궁화호 객차의 개조목록을 보면 일부도색이 통일호 객차와 겹치는 것으로 알고 있습니다. 제 개인적인 생각으로는 2안으로 가되 통일호 객차와 중복되는 도색을 무궁화호 객차에서 빼야 한다고 생각합니다

첫 글 끝 부분에도 적었다시피 그것은 한열셋 객차가 도입년도나 성능이 아니라 여객등급별로 나뉘어져있는 특징으로 인한 문제입니다. 현재 한열셋 객차에는 말씀하신 무궁화호와 통일호에 동시에 있는 객차 뿐만아니라 새마을호와 무궁화호, 통일호와 비둘기호에 동시에 있는 객차또한 있습니다. 

제가 여기서 주장하는 바는 이걸 전부 뜯어고치기는 현실적으로 근시일 내엔 불가능하니 일단 냅두고, 지금 통일호 객차 혼자 등장년도가 84년으로 미뤄져있는 것은 불합리하니 최소한 이것만은 이전 63년 등장으로 되돌려야 한다는 뜻입니다.  **(2안)**

**Comment by telk5093** [comment]
(**2안**으로 변경하는 것을 긍정적으로 검토)

**Comment by telk5093** [comment]
d98c9c470051d2eefe9b08b01eeeee66854cc0fd 에서 2안으로 반영되었습니다.
